daft/series.py

```diff
@@ -1,27 +1,27 @@
 from __future__ import annotations
 
-from typing import Any, Literal, TypeVar
+from typing import TYPE_CHECKING, Any, Literal, TypeVar
 
 import pyarrow as pa
 
 from daft.arrow_utils import ensure_array, ensure_chunked_array
 from daft.daft import CountMode, ImageFormat, ImageMode, PySeries, image
 from daft.datatype import DataType
+from daft.lazy_import import LazyImport
 from daft.utils import pyarrow_supports_fixed_shape_tensor
 
+if TYPE_CHECKING:
+    import pandas as pd
+
+
 _NUMPY_AVAILABLE = True
```

Comment:
should we also do numpy to stay consistent with this pattern

Reply:
Sounds good. I hadn't removed numpy at the time because pyarrow was still pulling it in as a dependency. Now that we import pyarrow lazily, we can also import numpy lazily